middlewares/validators/task-requests.js

```diff
@@ -40,6 +41,61 @@ const postTaskRequests = async (req, res, next) => {
   }
 };
 
+const getTaskRequests = async (req, res, next) => {
+  const schema = joi
+    .object()
+    .keys({
+      dev: joi.bool().optional().sensitive(),
+      prev: joi.string().optional(),
+      next: joi.string().optional(),
+      size: joi.number().integer().positive().min(1).max(100).optional(),
+      q: joi.string().optional(),
+    })
+    .without("prev", "next")
+    .with("prev", "size")
+    .with("next", "size");
```

Comment:
Nit: 
```
.without("prev", "next")
.with("prev", "size")
.with("next", "size");
```

What do you mean by above 3 line?

Reply:
https://joi.dev/api/?v=17.9.1#objectwithkey-peers-options

Reply:
Understand Thank you